pkg/api/grpc/echo.go

```diff
@@ -0,0 +1,20 @@
+package grpc
+
+import (
+	"context"
+	"log"
+
+	"github.com/stefanprodan/podinfo/pkg/api/grpc/echo"
+	"go.uber.org/zap"
+)
+
+type echoServer struct {
+	echo.UnimplementedEchoServiceServer
```

Comment:
Why is this call Unimplemented?

Reply:
That's something protoc-gen-grpc-go compiler now throws an error for in newer version `must be embedded to have forward compatible implementations.`, it act as a placeholder or a default behavior for any methods that have not yet been implemented in your code